src/Integration.Vsix/SonarLintDaemon/DaemonCommands.vsct

```diff
@@ -18,6 +18,20 @@
           <ButtonText>Disable SonarLint rule</ButtonText>
         </Strings>
       </Button>
+
+      <Button guid="guidDaemonCmdSet" id="cmdidCreateCFamilyReproducer" priority="0x0200" type="Button">
+        <!-- It looks like the command needs to belong to a parent menu group, even though it only
+             appears in the command window.
+             We're adding it to the top-level Analye menu, so the command the user types will be
+               "Analyze.SonarLint.CFamily.Reproducer" -->
+        <Parent guid="guidCodeAnalysisMenuGroup" id="IDG_CODEANALYSIS_TOPLEVEL"/>
+        
+        <CommandFlag>CommandWellOnly</CommandFlag>
+        <Strings>
+          <ButtonText>SonarLint.CFamily.Reproducer</ButtonText>
```

Comment:
Perhaps the name should actionable, i.e. RunReproducer?

Reply:
Ok.